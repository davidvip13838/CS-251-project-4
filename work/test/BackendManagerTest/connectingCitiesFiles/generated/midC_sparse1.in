500
3886 8057
5590 8905
8340 5075
6312 1327
7267 4098
5482 6884
8558 6945
7357 3480
1502 8276
6217 607
8411 2402
514 7654
4623 5537
5920 9387
6401 6738
4499 1734
4784 379
5570 1210
3565 5288
1869 265
7580 446
353 529
6983 8745
1262 5519
6164 5266
9998 1631
1535 8916
5398 5078
7658 2383
8041 3486
5671 4630
2185 552
6779 571
3830 9415
1736 9335
7792 8149
8062 4205
5024 7803
448 4575
2534 7630
7088 8709
9188 9258
9741 8976
4672 6492
3842 3381
5355 23
4929 5175
2438 3709
6535 6022
8762 3312
7766 6106
8684 8204
2942 9635
2588 928
2345 6931
1790 5011
4378 2111
6777 3132
8531 8199
5533 8524
3612 1612
5837 3372
9603 1346
7639 3172
1309 2658
931 1707
2066 3891
1402 684
8449 8413
6443 9979
9160 5053
5769 7513
9702 9323
3235 1418
4740 4625
1266 9408
6277 7994
4924 6534
8682 6361
4864 930
9391 2515
3410 6407
8439 490
6455 5764
4495 1373
5804 9158
355 3763
5361 4843
4784 9660
7678 5556
4434 7815
4427 4180
3278 1401
9281 3167
8581 8306
727 8962
8736 9389
7014 1885
4801 5705
4625 9662
5881 438
2770 9885
4474 2617
3619 5554
9173 201
747 8214
1571 4777
995 538
5802 7421
849 9577
4018 3716
8296 7468
8364 6953
7068 8272
2313 4326
7607 3396
6221 4304
625 7730
4432 5369
3261 9599
4175 8070
1263 5035
8826 205
5343 8346
9103 8752
3937 5526
8835 2533
2343 7829
1281 887
6696 2082
7177 8229
8609 1112
330 2627
6479 2819
3034 7119
3639 5417
4860 9752
8078 6254
3576 6087
8586 4473
3535 2803
4655 837
6126 7153
3286 408
1297 2767
6442 8143
3664 456
4005 7306
1924 1823
2844 223
8951 7189
3818 6087
7723 460
8023 9351
7498 6041
5292 6882
953 3270
4908 131
8199 5794
9637 3588
2865 1145
3265 8639
4418 3780
2492 8874
3737 8593
1837 5434
9568 7294
618 9654
5103 3872
4562 6629
4917 1306
2093 5082
9852 7248
5697 51
4002 2873
9963 8448
3063 3330
4894 4468
9733 632
8277 3313
6276 7943
4757 7924
8652 7653
8305 7421
5826 1116
1060 4265
2728 7086
4025 6789
4363 3829
2162 6346
9732 6167
5369 5963
717 5734
2840 4870
3337 4201
6868 9699
3239 6197
7984 31
1681 6428
1584 245
527 7579
4009 8637
3750 3377
5862 3599
3305 9244
8030 3571
9688 908
7422 6150
100 4587
5566 8979
2404 9443
1468 8294
1822 1110
7545 5632
978 8285
676 586
1486 9098
6742 2423
1076 4202
8891 2880
9146 3463
9506 7078
322 7404
5381 9820
3781 2157
6347 581
6215 2012
1735 1650
2275 5931
2941 6290
7664 3997
8426 1818
7784 8887
6641 7116
109 9135
3991 7300
4100 9466
343 8667
5928 1574
1780 3896
3619 797
3960 2818
5330 5371
9072 9966
8213 9476
706 2986
1142 9720
2112 4304
3026 2343
5731 1470
3244 9089
7580 4218
6806 9720
9649 1737
8744 2458
3350 6242
585 6731
1898 5828
2691 2313
9338 3992
4097 9988
3576 8294
1992 7798
6770 2070
8054 8664
6605 126
9548 1814
6719 1002
6643 6739
5344 3872
9518 7326
6404 111
8560 4969
4332 2377
1628 1880
6437 7396
9615 4253
6207 242
7689 6162
4729 6809
6241 2092
4132 4050
5376 586
7230 859
9630 9922
2271 7745
5858 922
7035 5849
6398 4707
9108 7886
4866 9104
109 8403
6446 8198
680 1271
2879 9908
420 5286
7755 7750
7398 2926
7775 2130
7112 4939
4729 5326
1459 6963
6886 2476
5114 7545
7779 8729
8804 6001
2410 4606
1207 9962
3709 7023
9843 9155
9907 3318
3858 1432
8720 7498
5347 8354
6123 8552
2333 3164
3023 2346
8060 4044
7624 5632
7930 3299
3667 6030
1542 23
1415 1683
4090 7081
4306 9499
114 9373
2779 6691
9163 6324
3285 2240
4619 3313
8956 2834
392 3968
4118 9229
1662 8087
4016 7119
215 4334
3099 9203
5810 5103
460 2426
3832 9287
7318 9659
5176 5844
2711 1229
6240 234
2364 4931
2552 7309
4710 3904
2191 8312
7838 6920
8169 6729
4330 2389
1969 1615
8546 9972
4485 461
7413 8568
9100 8442
3592 9314
4316 8026
4599 5267
7997 9719
8780 9661
5813 3397
6687 2073
1122 9448
8626 319
4691 2191
9109 543
3276 3696
2912 6940
9266 4782
6996 6753
4913 1676
191 2023
9242 5953
9843 3443
783 5611
332 9524
1848 5331
8568 4289
2241 1736
2766 2901
1010 122
8941 1281
5280 7314
1702 8822
23 3948
6039 8225
978 2731
3086 2323
9822 8216
8576 1443
448 9527
6712 7057
4877 3626
2121 9859
2789 155
7403 7640
5600 3488
9637 3166
8454 5837
4286 5785
1976 7155
640 1886
9803 6814
6066 6926
3359 1327
69 3763
4705 5684
1541 6888
2511 3364
2925 8468
7336 8059
4824 8655
9588 5345
326 6160
9632 6407
4130 2589
1860 3415
1178 6879
6364 315
2528 2262
3926 9415
6240 9572
7888 158
1946 9163
1812 7224
5915 6258
9899 6521
8336 3264
6536 6129
9351 110
2342 8006
5556 9898
3052 9538
9667 7841
5364 446
5620 5351
6139 5478
6354 4511
9918 5580
7571 8568
5981 4429
1055 8014
4285 3042
2046 1773
6514 4488
1217 2068
7632 5718
9224 8765
8444 8752
7785 6872
4056 8823
1613 1628
9081 6117
5670 1036
9890 4910
401 4021
3764 3166
3784 8427
7973 5342
3821 5718
2187 9443
4066 5150
8352 9912
8413 9596
8175 3343
8835 1140
5725 311
6518 2497
4887 5076
6254 9259
714 1528
2565 6732
8108 9909
987 3794
7760 8809
3952 1478
3407 9039
7404 5000
2113 8188
1232 8345
4447 1625
2103 1711
8343 3831
1254 6119
8038 8132
5957 159
8069 4418
6045 9312
4369 6091
5865 1460
9546 9156
742 3476
1115 5841
2176 5170
250 7083
96 1123
2707 7273
9059 8191
8040 8250
