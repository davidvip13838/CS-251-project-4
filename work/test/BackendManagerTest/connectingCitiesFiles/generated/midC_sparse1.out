148742.62045699023